samcli/commands/local/lib/generated_sample_events/events/sagemaker/PreHumanTask.json

```diff
@@ -0,0 +1,7 @@
+{
+    "version": "2018-10-16",
+    "labelingJobArn": "arn:{{{partition}}}:sagemaker:{{{region}}}:{{{account_id}}}:labeling-job/{{{labeling_job_name}}}",
+    "dataObject" : {
+        "source-ref": "s3://{{{bucket}}}/{{{data_object_key}}}"
```

Comment:
Why split out bucket and data_object_key in this way? Might be easier for customers to provide the S3 uri directly.

Reply:
See below